app/src/main/java/com/univoice/data/dto/request/RequestDepartmentDto.kt

```diff
@@ -0,0 +1,10 @@
+package com.univoice.data.dto.request
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class RequestDepartmentDto(
+    @SerialName("universityName")
+    val universityName: String
```

Comment:
P2 : 혹시나 null 데이터가 들어올 경우를 대비해서 초기 값 넣어주는게 좋아용
```suggestion
    val universityName: String = ""
```

Reply:
반영완!